openflow13/match.go

```diff
@@ -483,7 +485,84 @@ func NewVlanIdField(vlanId uint16) *MatchField {
 	vlanIdField.VlanId = vlanId | OFPVID_PRESENT
 	f.Value = vlanIdField
 	f.Length = uint8(vlanIdField.Len())
+	
+	if mask == true { 
+		mask := new(VlanIdField)
+		mask.VlanId = OFPVID_PRESENT
+		f.Mask = mask
```

Comment:
This is not correct. Please see NewEthSrcField() for an example of how to use points to pass the mask